types/markdown-it/lib/helpers/parse_link_label.d.ts

```diff
@@ -0,0 +1,5 @@
+import State = require('../rules_core/state_core');
+
+declare function parseLinkLabel(state: State, start: number, disableNested?: boolean): number;
```

Comment:
I think there is a bug here. The `state` could also be `StateInline` here. Could you check it?

Reply:
@plantain-00 @duduluu